tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
 	"compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```

Comment:
It is for the examples. I would hate to enable it as you know it would be problematic.
But the main typings does not rely on it.